site/_apps/elm-hn-pwa.txt

```diff
@@ -0,0 +1,25 @@
+---
+weight: 24 
+title: 'Elm HN PWA'
+github-title: 'skrypte/elm-hn-pwa'
+libraries: "Elm" 
+scaffolding: 'N/A'
+module-bundling: 'N/A'
+service-worker: 'partially workbox.js + application shell + data caching'
```

Comment:
Could you describe why the entry says 'partially' for workbox support? Generally we keep this fairly high-level (e.g uses workbox + application shell + data-caching)

Reply:
workbox is used only as static cache for assets and application shell